woocommerce-gateway-paypal-express-checkout.php

```diff
@@ -36,7 +36,7 @@ function wc_gateway_ppec() {
 	if ( ! isset( $plugin ) ) {
 		require_once( 'includes/class-wc-gateway-ppec-plugin.php' );
 
-		$plugin = new WC_Gateway_PPEC_Plugin( __FILE__, '1.2.0' );
+		$plugin = new WC_Gateway_PPEC_Plugin( __FILE__, '1.2.1' );
```

Comment:
Perhaps we should define a global version as per our standardization so that we don't need to change versioning in multiple places?